zebra/zebra_vty.c

```diff
@@ -948,6 +948,38 @@ DEFUN (show_ip_nht,
 	return CMD_SUCCESS;
 }
 
+DEFPY (show_ip_import_check,
+       show_ip_import_check_cmd,
+       "show <ip$ipv4|ipv6$ipv6> import-check [vrf NAME$vrf_name|vrf all$vrf_all]",
+       SHOW_STR
+       IP_STR
+       IP6_STR
+       "IP import check tracking table\n"
+       VRF_CMD_HELP_STR
+       VRF_ALL_CMD_HELP_STR)
+{
+	afi_t afi = ipv4 ? AFI_IP : AFI_IP6;
+	vrf_id_t vrf_id = VRF_DEFAULT;
+
+	if (vrf_all) {
+		struct vrf *vrf;
+		struct zebra_vrf *zvrf;
+
+		RB_FOREACH (vrf, vrf_name_head, &vrfs_by_name)
+			if ((zvrf = vrf->info) != NULL) {
+				vty_out(vty, "\nVRF %s:\n",
+					zvrf_name(zvrf));
+				zebra_print_rnh_table(zvrf_id(zvrf),
+						      afi, vty,
+						      RNH_NEXTHOP_TYPE);
+			}
+		return CMD_SUCCESS;
+	} else if (vrf_name)
```

Comment:
nit: this `else` can be removed as we'll return early when `vrf_all` is set (block above).